Dockerfile

```diff
@@ -65,6 +65,7 @@ RUN /tmp/install_deps/install_scancode.sh
 COPY ./hack/py23requirements.txt /tmp/install_deps/
 RUN pip2 install -r /tmp/install_deps/py23requirements.txt
 RUN pip3 install -r /tmp/install_deps/py23requirements.txt
+RUN pip2 install --egg git+https://github.com/gofed/gofedlib.git@4ca44e4
```

Comment:
Could you separate it by a blank line, since it AFAICT does not belong to the above 'required in both Py2&3` section ?

Otherwise LGTM so feel free to merge yourself.

Reply:
Blank line added 😉